tests/test_plugin.py

```diff
@@ -22,3 +23,15 @@ def test_anon(client):
     response = client.get(url_for('Airflow.index'))
     assert response.status_code == 302
     assert b"update-notice.css" not in response.data, "Don't show notice when logged out"
+
+
+def test_migrations_applied(session):
+    """Verify that the migrations are applied correctly"""
+    from astronomer.airflow.version_check.plugin import AstronomerVersionCheckPlugin
+    from astronomer.airflow.version_check.models import AstronomerAvailableVersion
+
+    plugins_manager.ensure_plugins_loaded()
+    AstronomerVersionCheckPlugin.migrate_db_tables()
```

Comment:
Don't we need the tables to be without these columns? I suspect if you comment this line out, the following assert would still pass?

Reply:
Removed it as this test is not valid now.